src/compiler/python_generator.cc

```diff
@@ -66,6 +66,9 @@ using std::vector;
 
 namespace grpc_python_generator {
 
+GeneratorConfiguration::GeneratorConfiguration()
```

Comment:
Another possibility is to have GeneratorConfiguration take 2 arguments.  I was just remarking on how we now have a config object that is not configurable.